src/config/power-management.md

```diff
@@ -17,7 +17,9 @@ and enabled. Either disable `acpid` when enabling `elogind`, or configure
 `elogind` to `ignore` ACPI events in
 [logind.conf(5)](https://man.voidlinux.org/logind.conf.5). There are several
 configuration options, all starting with the keyword `Handle`, that should be
-set to `ignore` to avoid interfering with `acpid`.
+set to `ignore` to avoid interfering with `acpid`. In order to be able to use
+`loginctl {poweroff,reboot}`as a normal user it is needed to install the polkit
+package.
```

Comment:
I think this is better added in a separate paragraph:

```suggestion
set to `ignore` to avoid interfering with `acpid`.

In order to be able to use `loginctl poweroff` and `loginctl reboot` as a normal user, install the `polkit` package.
```

Reply:
Also, I think I recall that `polkit` is actually required to run something like Sway at all?

Reply:
Thanks, sorry for the delay totally forgotten this.

> Also, I think I recall that `polkit` is actually required to run something like Sway at all?


I recall that also, not 100%, but I think that without polkit package installed and service running, it will not launch. (doesn't require a polkit client).